### PR TITLE
Term entry atanh.md

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Description

Term entry for kotlin.math.atanh()

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Kadif-Ravik** [comment]
uhg

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+Title: 'atanh()'
+Description: 'Calculates the inverse hyperbolic tangent of a value'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:3]:
```suggestion
Description: 'Calculates the inverse hyperbolic tangent of a value.'
```

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`atanh()`** method in Kotlin's `math` class returns the inverse hyperbolic tangent of any number between `-1.0` and `1.0` in radians.
+
+## Syntax:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:20]:
```suggestion
## Syntax
```

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+```pseudo
+import kotlin.math.*
+val y = Math.atanh(x)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:24]:
```suggestion
val y = math.atanh(x)
```

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+- `x`: A number between `-1.0` and `1.0`, of datatype `Double` or `Float`.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:27]:
```suggestion
- `x`: A number between `-1.0` and `1.0`, of type `Double` or `Float`.
```

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+## Example:
+
+The `math.atanh(x)` method returns the value `y` such that `tanh(y) == x`
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:31]:
```suggestion
The following code is a basic implementation of `math.atanh()`.
```

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+## Syntax:
+
+```pseudo
+import kotlin.math.*
+val y = Math.atanh(x)
+```
+
+- `x`: A number between `-1.0` and `1.0`, of datatype `Double` or `Float`.
+
+## Example:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:29]:
```suggestion
## Example
```

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+The returned value is `y` such that `tanh(y) == x`.
+
+### Special Cases:
+```psuedo
+  tanh(NaN) = NaN
+  tanh(x) = NaN if x > 1 || x < -1
+  tanh(1.0) = +Inf
+  tanh(-1.0) = -Inf
+```
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:58]: These lines should come after the syntax section and before the example. And generally we do not use a code block for the special cases

File: content/kotlin/concepts/math-methods/terms/atanh/atanh.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+The `math.atanh(x)` method returns the value `y` such that `tanh(y) == x`
+
+```kotlin
+import kotlin.math.*
+fun main() {
+  val x = -0.1
+  val y = atanh(x)
+
+  // Print results
+  println('$y')
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atanh/atanh.md:40]:
```suggestion
  println(y)
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Kadif-Ravik I've added my comments. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by Kadif-Ravik** [comment]
@caupolicandiaz I think i got it.

**Comment by Kadif-Ravik** [comment]
@caupolicandiaz

**Review by caupolicandiaz — APPROVED** [review]
@Kadif-Ravik I made a few additional direct edits. Looks good for a second review

**Comment by Kadif-Ravik** [comment]
@caupolicandiaz dunno how I overlooked the special cases. thanks

**Comment by Kadif-Ravik** [comment]
@caupolicandiaz @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@Kadif-Ravik Looks good.

**Pull request merged**